src/libs/E2E/apiMocks/signinUser.json

```diff
@@ -0,0 +1,123 @@
+{
+  "onyxData": [
+    {
+      "onyxMethod": "merge",
+      "key": "session",
+      "value": {
+        "authToken": "AF93359A02093FDC2CB3B7E3923F9E108E4E2B6FC294FF19364B7F1032FD32D9137D276FF4A829309ECBAEA6EE3FC1A2AD917ABCAA2F97C3E4069177F2D89DCB7C14512633E37874EE1040E2E5EAE3C593BF76AB4F7624B819EB38A835319C0ADCDC73BA58FC7343DEED3FF5A22D9C8D7CD1647D171D9935A7C5E224FB5BE5ECECE940F6309EEFD14DE4F77AC96D4EB53D00ACF3F223325955267B881537B78F5A6DD6236365663FF914DAA1BBB58614D2A665F1ABF4346A1011244290E74E6FE02C730F1EC8061C5D89A9A99BEAACCCFDA2C0AA386262F0589E0D390585784E9026931C644E69D6E7C1711577B97265C8F586F9E08E01429E6E23D14C01E6A1B3E5676D75BB7A479D214E913CF6A80F0EBA5B3400D6F9DB10C2C984B85A527E900BDCE599122B8897C65BF5A8078EF0EF485D6D2ECB5A41353D85C2A8E33A4082AFED36E843617B80608FBED9B98903",
+        "accountID": 12313081,
+        "email": "applausetester+perf2@applause.expensifail.com",
+        "encryptedAuthToken": "D0BOkwW8nBQK3cCEmAT15jGo6e/wwrZKfV87NwwywIuWzfF0esxz2wE9YV1jGq38VoH7wg04/wqDizA4BCC/17WooG8KBNgCVJuROnxrw4swepgskehB0QDIPfhe5tUOR0oQTsLDYOuxvh5dUW+dh+ZZ4bws8HG+BLWXCUyQRht/f/5xXKWP3NwWpPOunx+BJtkizt7B50Qrl43bK+sV+6/P7P5OuyqaX+1AOOcW+nmW0zaE/ig8+y5CRtcMiA/6qQJi4RhTW/43dZELcPkMHc1K8CcqwOVyjAU76ChKqWrrdRzESgD4BJMEqYw/i7k9XqsKqBoAL4b4bJqW6N6jQa7dPWL3cSbEsWlwQ98C1zUZ0h/BsQDc88L77TemVzv044QckBu5hJvkx74ek16n29wHlLJMt5m6EGuvgHI72F1qrrdeQpDdS/9gEVyMTfq9vl1fada8EO5v+hbeYdoFuJK7/MKQrPXiYYtzEbVfHKtNlzzb6jLDTOL+2/Q3rOPBts0+gh6pvsBTmTjeFI9Afy2aF6r6c0LE9P3p8rJNfhUdhKlnSbcKEZPJYgFPxeTxSoZ4tNM42ae4Dgg0pL30w4xlCgIe9eAg3bQkLfWpax8VTwEIhelk0coDkXRijFZ+ZMfMM3ceZD8V6YyCBMjaQIOlovv6wWCgqGGsyj0yx2zVhrs7in8NU3wsujTYJA4z+pyBvfVEv7X/gDOxQvtGzbKt19Gk94FTDip4FYe0E5NdGcFnxjm5aosLZ0JruY4pZQaE/sW63vVAxRQLbpj2sqJmrN1xCOQF5ZPaBf7s9wz5uUejDyhmT8nzyEYOwnBqsbm6q5jnZjCQwRwtH7ercvuRFwDFdMsS18bvslCO8+YfFN1/r18tqsbpOtS71DOWwlR8JoeNIHi3wvYQIz3gcgWzXvcpEUnhYU4wl3D2D7u0untoAV44GEzPNpHzkBzr15eQXLiv6N/Gn2FGIWmTzQ=="
```

Comment:
These auth tokens should probably be removed too and just replaced with something fake